client/src/components/Container.jsx

```diff
@@ -24,12 +24,11 @@ class Container extends React.Component {
     return (
       <div style={styles.container}>
         <h2 style={styles.header}>Reviews</h2>
-        <ReviewContainer />
-        <ReviewContainer />
-        <ReviewContainer />
-        <ReviewContainer />
-        <ReviewPhotosContainer />
-        <button onClick={this.toggleModal}>Modal</button>
+        <ReviewContainer showPrice="false"/>
+        <ReviewContainer showPrice="false"/>
+        <ReviewContainer showPrice="false"/>
+        <ReviewContainer showPrice="false"/>
+        <ReviewPhotosContainer openModal={this.toggleModal}/>
         <Modal showModal={this.state.showModal} onClose={this.toggleModal}/>
```

Comment:
what does `onClose` do? it's not in the react docs and there's sparse info online